dashboard/UI-Test/selenium_drivers/README.md

```diff
@@ -0,0 +1,9 @@
+Driver version currently shipped:
+- Chrome/Chromium = v105.0.5195.19
+- Firefox(gecko) = v0.31.0
+
+> NOTE:
+- Selenium_drivers folders have your selenium webdrivers geckodrivers(for firefox) and chromedrivers(for chrome and chromium)
+- If script fails due to drivers issue, you need to find appropriate selenium webdriver according to your browser version(choose latest if your browser is upto to date)
```

Comment:
This statement seems self-evident, so I would either omit it or rephrase it along the lines of, "You must provide a Selenium webdriver which matches your choice of browser and browser version."